Patch sent to serf-dev@googlegroups.com, applied upstream in r1223.

--- a/buckets/ssl_buckets.c
+++ b/buckets/ssl_buckets.c
@@ -619,8 +619,8 @@
 }
 
 #if APR_HAS_THREADS
-apr_pool_t *ssl_pool;
-apr_thread_mutex_t **ssl_locks;
+static apr_pool_t *ssl_pool;
+static apr_thread_mutex_t **ssl_locks;
 
 typedef struct CRYPTO_dynlock_value {
     apr_thread_mutex_t *lock;
